mkosi/__init__.py

```diff
@@ -823,6 +833,62 @@ def install_systemd_boot(state: MkosiState) -> None:
                      state.workspace / "mkosi.esl"])
 
 
+def install_shim(state: MkosiState) -> None:
+    if not want_efi(state.config):
+        return
+
+    if state.config.shim_bootloader == ShimBootloader.none:
+        return
+
+    if not any(gen_kernel_images(state)) and state.config.bootable == ConfigFeature.auto:
+        return
+
+    signed = [
+        "usr/lib/shim/shim*.efi.signed.latest",
+        "usr/lib/shim/shim*.efi.signed",
+        "boot/efi/EFI/*/shim[a-z][0-9]*.efi",
```

Comment:
I'm not quite sure what this is supposed to match, a comment with examples would probably be helpful saying which cases occur where, since this seems to mix glob and regex semantic, e.g. this would match a file `shima9%%.efi`, because the star doesn't affect the character ranges before it. Is that intended?

Reply:
Fixed in the latest push